service/src/event/service.rs

```diff
@@ -237,6 +241,168 @@ impl CeramicEventService {
         })
     }
 
+    // We map over this to get a vec of conclusion events
+    fn transform_raw_events_to_conclusion_events(
+        &self,
+        raw_event: ceramic_api::EventDataResult,
```

Comment:
As noted, we should not use the ceramic_api types here but instead use the `unvalidated::Event` directly.